app/views/support_interface/support_users/index.html.erb

```diff
@@ -37,6 +38,11 @@
         <td class='govuk-table__cell'>
           <%= support_user.dfe_sign_in_uid %>
         </td>
+        <td class='govuk-table__cell'>
+          <%= link_to(support_interface_confirm_destroy_support_user_path(support_user)) do %>
```

Comment:
We should use a `govuk-link`, there's a helper called `ViewHelper#govuk_link_to` for this. 😉 

```suggestion
          <%= govuk_link_to(support_interface_confirm_destroy_support_user_path(support_user)) do %>
```

Reply:
@chubberlisk I think for this to work with a block we would need to change `govuk_link_to` to have a similar method signature to `link_to`

```ruby
 def link_to(name = nil, options = nil, html_options = nil, &block)
```
For the sake of expediency I'll add the `govuk-link` css class to the html_options.

Reply:
Ah yush! Sorry, I didn't realise that it was a block. 🤦‍♀ 

Makes sense, do that.